.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+
+    STOP! If you have questions about the RFC process or whether
+    your idea is of interest to the ESLint team, please send an
+    email to the mailing list instead of opening an issue:
+
+        https://groups.google.com/group/eslint/
+
+    As an alternative, you can also visit the chatroom to discuss
+    your idea:
+
+        https://gitter.im/eslint/eslint/
+
+    Thanks in advance for your help.
+
+-->
```

Comment:
If we don't want people to file issues on this repo, should we just disable issues?

Reply:
Oh duh! Of course we should.